ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -17,17 +39,6 @@ else
 		fmu i2c 2 100000
 	fi
 
-	if ver hwcmp PX4FMU_V4
```

Comment:
I'm not sure if we can move this

Reply:
Sounds good.  I've reverted that and reduced the number of changes in rc.sensors so that there is a little less to check in this PR.  If it appears we can move this into the `if ver hwcmp PX4FMU_V4` further down we can do that in a subsequent PR, (perhaps as its' own PR with just that change).  Thanks!